eclipse-theia/theia-blueprint
# Theia 1.49.1, Dev Containers, and Splash Screen

#### What it does

* update to latest Theia
* enable the dev containers feature
* make use of the new splash screen feature
* update built-ins to 1.88.1
* disable splashscreen in AUT for now

Contributed on behalf of STMicroelectronics

#### How to test
Build and check if everything is still working. 
Also check if Splash Screen is visible on startup and that dev containers feature is available and working

#### Review checklist


#### Reminder for reviewers

- as a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)
